data/talks.yml

```diff
@@ -0,0 +1,125 @@
+talk_list:
+  - timeslot: "09:00–09:30"
+    title: "Intro"
+    abstract: ""
+    slug: "intro"
+    speaker:
+      name: ""
+      avatar: ""
+  - timeslot: "09:30–10:00"
+    title: "Understanding 'Hello World'"
+    abstract: "'Hello, World!' is the first step of many a ruby journey. How much can we learn by going deep on everything it can teach us? Join me on a fun dive down through the stack as we go deep on everything from Strings, IO, Unix, the Ruby VM and x86 Assembler letting the simplest of programs be our guide."
```

Comment:
Rather than duplicating these talks, titles & abstracts - how about we somehow cross reference them? So:

```
#talks.yml (maybe this should even be schedule.yml)
schedule_items:
  - timeslot: "08:30-09:00"
    title: "Registration"
    description: "Pick up your nametag and mingle at Te Papa."
    slug: "rego"
  - timeslot: "09:00-09:30"
    title: "Intro"
    description: "Introduction to the conference."
    slug: "intro"
  - timeslot: "09:30–10:00"
    talk: "eoin"
    slug: "eoin"
```

Then in the view (although this much Ruby should probably go somewhere else...)

```
schedule_items.each do |schedule_item|
  .c-talk id=schedule_item.slug
    .c-talk__timeslot.col-1-4 = schedule_item.timeslot
    .c-talk__detail.col-3-4
      - if timeslot.talk
        - talk = data.speakers.speaker_list.select { |speaker| speaker.slug == timeslot.talk }
          h2.c-talk__title = talk.title
          p.c-talk__abstract= talk.abstract
          .c-talk__speaker
             img.c-talk__speaker-avatar src=("/images/speaker-avatars/#{talk.speaker.avatar}") alt=(talk.speaker.name)
          h3.c-talk__speaker-details 
            a href="/speakers/##{talk.slug}"
              = talk.speaker.name 
      - else
        h2.c-talk__title = schedule_item.title
        h2.c-talk__abstract = schedule_item.description
```

Then you've got the speaker data from the speaker data file and we don't have to double up on anything

Reply:
@mermop Thanks for the suggestion. I've changed around the yaml structures, and with Tim's help have refactored the view to pull data from `speakers` and `schedule`.